entity/c/load_config.h

```diff
@@ -0,0 +1,43 @@
+#ifndef LOAD_CONFIG_H
+#define LOAD_CONFIG_H
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define MAX 256
+#define ENTITY_INFO_NAME 1
+#define ENTITY_INFO_PURPOSE 2
+#define ENTITY_INFO_NUMKEY 3
+#define AUTH_INFO_PUBKEY_PATH 4
+#define ENTITY_INFO_PRIVKEY_PATH 5
+#define AUTH_INFO_IP_ADDRESS 6
+#define AUTH_INFO_PORT 7
+#define ENTITY_SERVER_INFO_IP_ADDRESS 8
+#define ENTITY_SERVER_INFO_PORT_NUMBER 9
+#define NETWORK_PROTOCOL 10
+
+typedef struct {
+    unsigned char name[32];
+    unsigned char purpose[32];
+    int numkey;
+    unsigned char *auth_pubkey_path;
+    unsigned char *entity_privkey_path;
+    unsigned char auth_ip_addr[17];
+    unsigned char auth_port_num[6];
+    unsigned char entity_server_ip_addr[17];
+    unsigned char entity_server_port_num[6];
+    unsigned char network_protocol[4];
+} config_t;
```

Comment:
We need a free function for config_t, as it contains data created from malloc, such as *auth_pubkey_path, and *entity_private_path.

Reply:
This is in `c_api.c` `free_SST_ctx() `

Reply:
Here is the change I meant, for your information in the future: https://github.com/iotauth/iotauth/pull/12/commits/6cc01ad42b333b3a1687e4e2e974d89a531f663a